### PR TITLE
feat: show archived status on branches

### PR DESCRIPTION
**Pull request by xenomote**
add `current_state` to branch list to show archived state
remove `updated_at` from the branch list, because it is an internal detail 

image

### CODE REVIEW

File: src/commands/branches.ts [hunk 1]
```diff
@@ -21,18 +21,20 @@ import { log } from '../log.js';
 import { parseSchemaDiffParams, schemaDiff } from './schema_diff.js';
 import { getComputeUnits } from '../utils/compute_units.js';
 
-export const BRANCH_FIELDS = [
+export const BRANCH_FIELDS: readonly (keyof Branch)[] = [
```
Comment (duskpoet) [inline src/commands/branches.ts:24]: you can remove the `as const` modifier then

**Comment by duskpoet** [comment]
it doesn't fit my screen anymore :( 
image

**Comment by duskpoet** [comment]
I'd suggest getting rid of updated_at column, I don't think it's that important (it's not the same as last active, right?)

**Comment by xenomote** [comment]
> it's not the same as last active, right?

No just when any detail of the branch was updated. Last active is a property of endpoints not branches

**Review by amitdahan — APPROVED** [review]
LGTM

**Comment by amitdahan** [comment]
(apart from @duskpoet 's concerns about width? maybe we can do something about it?)

**Comment by xenomote** [comment]
> maybe we can do something about it?

removing the `updated_at` column probably fixes that, it wasn't really supposed to be a user facing field anyway

**Pull request merged**